learning_journal/views/default.py

```diff
@@ -25,8 +25,7 @@ def write(request):
         new_title = request.POST["title"]
         new_body = request.POST["body"]
         new_date = time.strftime("%m/%d/%Y")
```

Comment:
If you changed the `creation_date` field to a DateTime object, you could just do `datetime.now()` to get the current time.